### PR TITLE
Add profiling service page tab example

### PR DESCRIPTION
**Pull request by gturbat**
### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/profiler/connect_traces_and_profiles.md [hunk 1]
```diff
@@ -322,11 +322,23 @@ With endpoint profiling you can:
...
+```
+
+{{< img src="profiler/apm_service_page_pivot_to_contention_comparison.mp4" alt="Pivoting from APM service page to Profiling comparison page to find the line of code causing latency" video=true >}}
+
+### Track endpoints that consume the most resources
 
 It is valuable to track top endpoints that are consuming valuable resources such as CPU and wall time. The list can help you identify if your endpoints have regressed or if you have newly introduced endpoints that are consuming drastically more resources, slowing down your overall service.
 
-The following image shows that `GET /store_history` is periodically impacting this service by consuming 20% of its CPU:
+The following image shows that `GET /store_history` is periodically impacting this service by consuming 20% of its CPU and 50% of its allocated memory:
```
Comment (maycmlee) [inline content/en/profiler/connect_traces_and_profiles.md:341]: Could you change the name of the new image? There's a caching issue when we use the same image name, the new image won't show up.

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

File: content/en/profiler/connect_traces_and_profiles.md [hunk 2]
```diff
@@ -322,11 +322,23 @@ With endpoint profiling you can:
...
+
+In APM Service page, use `Profiling` tab to correlate a latency/throughput change to a code performance change.
+
+In this example, you can see how latency is linked to a lock contention increase on `/GET train` caused by the following line of code
+
+```java
+Thread.sleep(DELAY_BY.minus(elapsed).toMillis());
+```
+
+{{< img src="profiler/apm_service_page_pivot_to_contention_comparison.mp4" alt="Pivoting from APM service page to Profiling comparison page to find the line of code causing latency" video=true >}}
```
[the remark this answers is not in the document]
Comment (gturbat) [reply content/en/profiler/connect_traces_and_profiles.md:335]: I used the new nav with this is mind, is it fine?

File: content/en/profiler/connect_traces_and_profiles.md (hunk 1 above)
Comment (gturbat) [reply content/en/profiler/connect_traces_and_profiles.md:341]: Done in https://github.com/DataDog/documentation/pull/22475/commits/d812efe47fcbab3f60e88a8dad22fb5a63af1406

File: content/en/profiler/connect_traces_and_profiles.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (maycmlee) [reply content/en/profiler/connect_traces_and_profiles.md:335]: Oh sorry, I missed that. Do you mind making the video without the nav? We will be removing the nav from the older videos.

[the remark this answers is not in the document]
Comment (gturbat) [reply content/en/profiler/connect_traces_and_profiles.md:335]: done in https://github.com/DataDog/documentation/pull/22475/commits/1617384785d05a66e9b263dec40c5bc1311151b4

**Review by maycmlee — APPROVED** [review]

**Pull request merged**